tutorials/oauth2-with-23andMe-in-java/index.md

```diff
@@ -0,0 +1,17 @@
+This sample app uses Google's OAuth 2.0 java lib to authenticate with 23andMe.  Once authenticated the app pulls a single genotype for i3003137, a SNP associated with Sickle Cell Anemia.
+ 
+### Instructions
+* Create a 23andMe client at https://api.23andme.com/apply/ 
+    * set your redirect_uri of http://127.0.0.1:8080/Callback
+    * you will be given a client_id and a client_secret
```

Comment:
consider using backticks sytax to render them as `client_id` and `client_secret`